javascriptv3/example_code/README.rst

```diff
@@ -0,0 +1,43 @@
+###############################
+AWS SDK for JavaScript version 3 (v3) examples
+###############################
+
+These are examples for the AWS SDK for JavaScript (v3) public documentation.
+
+The prerelease version of the version 3 of the SDK is available `here <at https://github.com/aws/aws-sdk-js-v3>`_ .
+The 'SDK for JavaScript Developer Guide' for V3 is also scheduled for release later in 2020, and the topic containing this example will be hosted
+`here <https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sqs-examples-managing-visibility-timeout.html`_ .
+
+**NOTE:** The AWS SDK for JavaScript (v3) is written in TypeScript so, for consistency, these examples are also in TypeScript. TypeScript extends of JavaScript so these examples can also be run as JavaScript.
```

Comment:
Would it be worth a sentence or two explaining why TypeScript?

Reply:
The SDK team are confident average users will understand the difference. However, I've added a link to the TypeScript home page to each README